lib/perl/Genome/Config/AnalysisProject/Command/DiskGroupUsage.pm

```diff
@@ -0,0 +1,85 @@
+package Genome::Config::AnalysisProject::Command::DiskGroupUsage;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::Command::DiskGroupUsage {
+    is => 'Command::V2',
+    has => [
+        analysis_projects => {
+            is                  => 'Genome::Config::AnalysisProject',
+            doc                 => 'the analysis projects to report disk usage',
+            is_many             => 1,
+            shell_args_position => 1,
+        },
+        print_allocation_ids_for_disk_group_name => {
+            is => 'Text',
```

Comment:
This could be `print_allocation_ids_for_disk_group` (`is => 'Genome::Disk::Group'`) so users can identify disk group objects by any means they like.  (We would want to make sure that supplying a bare name would still find the object like it does for models.  I think we might need to give it a little help since the property is not currently called `name`, but we could add an alias.)

Reply:
I could use some help resolving the object by name:
`Resolving parameter 'print_allocation_ids_for_disk_group' from command argument 'prod-builder'... none found.`

Reply:
jasonwalker80/genome/pull/3 is one possible solution.